# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/urlbar/UrlbarInput.jsm
# Commit: 355b0329bd95
# Full Hash: 355b0329bd95d2a9573c04ece889c734da00f080
# Author: harry <htwyford@mozilla.com>
# Date: 2019-10-08 21:45:57
# Regressor Bug: 1563350
# File Overlap Count: 2
# Description:
#   Bug 1563350 - Add popover to the Touch Bar that displays when the Urlbar has focus. r=mikedeboer,spohl,fluent-reviewers,Pike,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D38563
# ==============================================================================

diff -r 653caa0c494a -r 355b0329bd95 browser/components/urlbar/UrlbarInput.jsm
--- a/browser/components/urlbar/UrlbarInput.jsm	Tue Oct 08 00:52:07 2019 +0000
+++ b/browser/components/urlbar/UrlbarInput.jsm	Tue Oct 08 00:52:07 2019 +0000
@@ -1774,6 +1774,8 @@
     if (this.getAttribute("pageproxystate") != "valid") {
       this.window.UpdatePopupNotificationsVisibility();
     }
+
+    Services.obs.notifyObservers(null, "urlbar-blur");
   }
 
   _on_click(event) {
@@ -1814,6 +1816,8 @@
     if (this.getAttribute("pageproxystate") != "valid") {
       this.window.UpdatePopupNotificationsVisibility();
     }
+
+    Services.obs.notifyObservers(null, "urlbar-focus");
   }
 
   _on_mouseover(event) {